### ISSUE TITLE
Laggy controlers and HMD 

### ISSUE THREAD

**Issue by Porazzh** [psmoveservice/PSMoveService#701]
So basicly i have 4 camera setup and I give them 30fps on everyone. but in SteamVR it seems to be like 10 or 5 fps because it's very laggy i use driver4vr and bandwitches 5 meters active without hub, is that a problem that I haven't got USB hub?

Oh and my psmoveservice console only says error like
libusb: error [windows_get_device_list] program assertion failed - existing device should share parent

pls help

**Comment by Porazzh** [comment]
Ps. In tracking pose everything looks good

**Comment by seene** [comment]
hm...have you tried to setup your cameras to 60 oder 75fps?

I get the libusb error too....but cameras and tracking are working fine. (i also have 5m active USB cable)

**Comment by Porazzh** [comment]
Yes i tried to but od i do that camera windows pop up but it is crashing immediatly

**Comment by Porazzh** [comment]
But if I connect only 2 cameras i can set 60 or even 75. I tried to change USB and it wasn't it :/ I think my bandwitches are just broken
